cms/src/components/groups/GroupEditor.vue

```diff
@@ -281,6 +281,22 @@ const duplicateGroup = async () => {
     });
 };
 
+const copyGroupId = (group: Group) => {
+    if (!group?._id) {
```

Comment:
I don't think this if is needed, because an id should always be present on the group.

Reply:
Yes ! i have to remove it now.